core/new-gui/src/app/app.module.ts

```diff
@@ -225,56 +215,45 @@ registerLocaleData(en);
         throwNoTokenError: false,
       },
     }),
-    MatTooltipModule,
-    CustomNgMaterialModule,
     BrowserAnimationsModule,
     NgbModule,
-    NgbPopoverModule,
     RouterModule.forRoot([]),
     FileUploadModule,
     FormsModule,
     ReactiveFormsModule,
     FormlyModule.forRoot(TEXERA_FORMLY_CONFIG),
     FormlyMaterialModule,
-    FormlyMatDatepickerModule,
+    OverlayModule,
+    MatInputModule,
+    MatTabsModule,
     NzDatePickerModule,
     NzDropDownModule,
     NzButtonModule,
     NzAutocompleteModule,
     NzIconModule,
     NzFormModule,
-    NzImageModule,
     NzListModule,
     NzInputModule,
     NzPopoverModule,
-    NzCalendarModule,
-    NzMenuModule,
-    NzMessageModule,
     NzCollapseModule,
     NzToolTipModule,
-    NzCheckboxModule,
     NzTableModule,
-    NzModalModule,
     NzSelectModule,
-    NzSliderModule,
     NzSpaceModule,
     NzBadgeModule,
     NzUploadModule,
-    NzNoAnimationModule,
```

Comment:
I thought we need this for testing purpose?

Reply:
It doesn't effect our tests, and it's not been used correctly.
Currently, it turns off all the animations off, and we don't want to turn off animation on our frontend completely. 

https://ng.ant.design/docs/animations/en